### ISSUE TITLE
Program runs without errors but the overlay doesn't appear

### ISSUE THREAD

**Issue by mpnordland** [yshui/index_camera_passthrough#4]
I built from a current checkout (cloned right before I built).
I used rust nightly to compile.

I'm using Arch Linux, I did have some packages out of date, but not ones relating to steam or graphics.
I have a NVIDIA GTX 1070 using proprietary drivers and I'm running steam with the native runtime.

I tried the menu button binding and the trigger binding. I have camera disabled in the SteamVR settings.
Steam and SteamVR were up to date. SteamVR version is 1.22.6 Beta. I couldn't find a version number for Steam.

I was using the "Stereo" display mode.

**Comment by yshui** [comment]
Can you post your config file? Can you add `debug = true` to the top of your config file, run the program manually, and post its console output here?

**Comment by mpnordland** [comment]
Hey, I went back and tried it yesterday and the first time I ran it, I started it before steam and my Steam VR stuff got kinda broken but I turned off the beta and did some fiddling and then once I got Steam VR working again, I launched the overlay with SteamVR working and the overlay worked!

I had a weird issue where it felt like one camera had a higher vantage point than the other and it made it really hard to look at stuff but overall a huge step forward. 

Thanks so much for getting this far on it!
